src/services/copyConfig/policies/policyOptions.js

```diff
@@ -132,7 +131,10 @@ class PolicyOptions extends Options {
     })
     this.#policy = policy
   }
-
+  addUrl() {
```

Comment:
If you add the branch when adding the policies options you might not need this method